build.bash

```diff
@@ -15,7 +15,9 @@ sha256sums=SHA256SUMS
 # also <https://github.com/ooni/probe-engine/issues/269>.
 assets_get_geoip() {
     echo "* Fetching geoip databases"
-    dbip_country_database_name=dbip-country-lite-2020-02.mmdb
+    year=`date "+%Y"`
+    month=`date "+%m"`
+    dbip_country_database_name=dbip-country-lite-$year-$month.mmdb
```

Comment:
This breaks if db-ip.com does not publish their database on the first day of the month.

Reply:
Look like it need db-ip.com's API to know if they publish new database. :-(

I take a look their [database update tool](https://db-ip.com/tutorials/database-import-update), that tool need an account and API key. Bad idea. I will close this PR.